examples/digital_fingerprinting/production/morpheus/dfp_duo_modules_inference.py

```diff
@@ -0,0 +1,308 @@
+                "lines": False, "orient": "records"
+            },
+            "cache_dir": cache_dir,
+            "filter_null": True,
+            "file_type": "JSON",
+            "schema": {
+                "schema_str": source_schema_str, "encoding": encoding
+            }
+        },
+        DFP_SPLIT_USERS: {
+            "module_id": DFP_SPLIT_USERS,
+            "module_name": "dfp_fsplit_users",
+            "namespace": MODULE_NAMESPACE,
+            "include_generic": da.include_generic,
+            "include_individual": da.include_individual,
+            "skip_users": da.skip_users,
+            "only_users": da.only_users,
+            "timestamp_column_name": config.ae.timestamp_column_name,
+            "userid_column_name": config.ae.userid_column_name,
+            "fallback_username": config.ae.fallback_username
+        },
+        DFP_ROLLING_WINDOW: {
+            "module_id": DFP_ROLLING_WINDOW,
+            "module_name": "dfp_rolling_window",
+            "namespace": MODULE_NAMESPACE,
+            "min_history": 1,
+            "min_increment": 0,
+            "max_history": duration,
+            "cache_dir": cache_dir,
+            "timestamp_column_name": config.ae.timestamp_column_name
+        },
+        DFP_DATA_PREP: {
+            "module_id": DFP_DATA_PREP,
+            "module_name": "dfp_data_prep",
+            "namespace": MODULE_NAMESPACE,
+            "timestamp_column_name": config.ae.timestamp_column_name,
+            "schema": {
+                "schema_str": preprocess_schema_str, "encoding": encoding
+            }
+        },
+        DFP_INFERENCE: {
+            "module_id": DFP_INFERENCE,
+            "module_name": "dfp_inference",
+            "namespace": MODULE_NAMESPACE,
+            "model_name_formatter": da.model_name_formatter,
+            "fallback_username": config.ae.fallback_username,
+            "timestamp_column_name": config.ae.timestamp_column_name
+        }
+    }
+
+    post_proc_config = {
+        "module_id": DFP_POST_PROCESSING,
+        "module_name": "dfp_post_processing",
+        "namespace": MODULE_NAMESPACE,
+        "timestamp_column_name": config.ae.timestamp_column_name
+    }
+
+    # Create a linear pipeline object
+    pipeline = LinearPipeline(config)
+
+    pipeline.set_source(MultiFileSource(config, filenames=list(kwargs["input_file"])))
+
+    # Here we add a wrapped module that implements the DFP Inference pipeline
+    pipeline.add_stage(
+        LinearModulesStage(config,
+                           module_config,
+                           input_port_name="input",
+                           output_port_name="output",
+                           output_type=MultiMessage))
+
+    pipeline.add_stage(MonitorStage(config, description="Preprocessing & Inference rate", smoothing=0.001))
+
+    pipeline.add_stage(
+        FilterDetectionsStage(config, threshold=2.0, filter_source=FilterSource.DATAFRAME, field_name='mean_abs_z'))
```

Comment:
Can we roll this into the inference module, or is there some reason to keep it outside?